test/test_format_az.py

```diff
@@ -0,0 +1,567 @@
+        self.assertEqual(pronounce_number(-21.234, places=3),
+                         "mənfi iyirmi bir nöqtə iki üç dörd")
+        self.assertEqual(pronounce_number(-21.234, places=4),
+                         "mənfi iyirmi bir nöqtə iki üç dörd")
+        self.assertEqual(pronounce_number(-21.234, places=5),
+                         "mənfi iyirmi bir nöqtə iki üç dörd")
+
+    def test_convert_hundreds(self):
+        self.assertEqual(pronounce_number(100), "yüz")
+        self.assertEqual(pronounce_number(666), "altı yüz altmış altı")
+        self.assertEqual(pronounce_number(1456), "min, dörd yüz əlli altı")
+        self.assertEqual(pronounce_number(103254654), "yüz üç milyon, "
+                                                      "iki yüz əlli dörd min, "
+                                                      "altı yüz əlli dörd")
+        self.assertEqual(pronounce_number(1512457), "bir milyon, "
+                                                    "beş yüz on iki min, "
+                                                    "dörd yüz əlli yeddi")
+        self.assertEqual(pronounce_number(209996), "iki yüz doqquz min, "
+                                                   "doqquz yüz doxsan altı")
+
+
+
+
+
+    def test_convert_scientific_notation(self):
+        self.assertEqual(pronounce_number(0, scientific=True), "sıfır")
+        self.assertEqual(pronounce_number(33, scientific=True),
+                         "üç nöqtə üç vurulsun on üstü bir")
+        self.assertEqual(pronounce_number(299792458, scientific=True),
+                         "iki nöqtə doqquz doqquz vurulsun on üstü səkkiz")
+        self.assertEqual(pronounce_number(299792458, places=6,
+                                          scientific=True),
+                         "iki nöqtə doqquz doqquz yeddi doqquz iki beş vurulsun "
+                         "on üstü səkkiz")
+        self.assertEqual(pronounce_number(1.672e-27, places=3,
+                                          scientific=True),
+                         "bir nöqtə altı yeddi iki vurulsun on üstü "
+                         "mənfi iyirmi yeddi")
+
+    def test_auto_scientific_notation(self):
+        self.assertEqual(
+            pronounce_number(1.1e-150), "bir nöqtə bir vurulsun "
+                                        "on üstü mənfi yüz əlli")
+        # value is platform dependent so better not use in tests?
+        # self.assertEqual(
+        #    pronounce_number(sys.float_info.min), "iki nöqtə iki iki times "
+        #                                          "on to the power of "
+        #                                          "mənfi üç yüz "
+        #                                          "və səkkiz")
+        # self.assertEqual(
+        #    pronounce_number(sys.float_info.max), "bir nöqtə yeddi doqquz "
+        #                                          "times on to the power of"
+        #                                          " üç yüz və səkkiz")
```

Comment:
I'd agree with the comment - this can be deleted unless you see specific value in it?

Reply:
yeah, most of these comments are the result of using the english files as template. I'll review and remove them.